contracts/StablecoinConverter.sol

```diff
@@ -182,18 +182,9 @@ contract StablecoinConverter is EpochTokenLocker {
         int[] memory tokenConservation = new int[](prices.length);
         for (uint i = 0; i < owners.length; i++) {
             Order memory order = orders[owners[i]][orderIds[i]];
-            require(order.validFrom <= batchIndex, "Order is not yet valid");
-            require(order.validUntil >= batchIndex, "Order is no longer valid");
-            // Assume for now that we always have sellOrders
```

Comment:
Has this been removed completely?

Reply:
nope. I even thought of you and made it a todo:

   // ToDo: implement logic also for buyOrders